hack/gencerts.sh

```diff
@@ -17,6 +17,61 @@
 # Generates a CA certificate, a server key, and a server certificate signed by the CA.
 
 set -e
+SCRIPT=`basename ${BASH_SOURCE[0]}`
+
+function usage {
+  cat<< EOF
+  Usage: $SCRIPT
+  Options:
+  -h | --help               	 This help info.
```

Comment:
`Display help information.`. And please make the line indented consistently with other lines.

Reply:
Now fixed. Thanks!

Reply:
Oh, I meant please also change the text to `Display help information.`.

Reply:
Ok. Now fixed too.